src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.ProjectState.cs

```diff
@@ -1,408 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Immutable;
-using System.Diagnostics;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Host;
-using Microsoft.CodeAnalysis.LanguageServer.Handler.Diagnostics;
-using Microsoft.CodeAnalysis.PooledObjects;
-using Microsoft.CodeAnalysis.Shared.Extensions;
-using Microsoft.CodeAnalysis.SolutionCrawler;
-using Microsoft.CodeAnalysis.Workspaces.Diagnostics;
-using Roslyn.Utilities;
-
-namespace Microsoft.CodeAnalysis.Diagnostics;
-
-internal partial class DiagnosticAnalyzerService
-{
-    private partial class DiagnosticIncrementalAnalyzer
-    {
-        /// <summary>
-        /// State for diagnostics that belong to a project at given time.
-        /// </summary>
-        private sealed class ProjectState
-        {
-            private const string SyntaxStateName = nameof(SyntaxStateName);
-            private const string SemanticStateName = nameof(SemanticStateName);
-            private const string NonLocalStateName = nameof(NonLocalStateName);
-
-            // project id of this state
-            private readonly StateSet _owner;
-
-            // last aggregated analysis result for this project saved
-            private DiagnosticAnalysisResult _lastResult;
```

Comment:
This was the cached data from teh last 'ForceProjectAnalysisAsync' call.  Since we don't need to cache that, this data went away.

Also note that if ForceProjectAnalysisAsync was never called, then this was *always* the "default/initial" value set in teh constructor which says "you can't use me because i don't actually have useful data in me".